packages/share_plugin/lib/share_plugin_method_channel.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:flutter/foundation.dart';
+import 'package:flutter/services.dart';
+
+import 'share_plugin_platform_interface.dart';
+
+
+class MethodChannelSharePlugin extends SharePluginPlatform {
+
+  @visibleForTesting
+  final methodChannel = const MethodChannel('share_plugin');
+
+
+  @override
+  Future<void> shareMethod(String message) async {
+    try {
+      await methodChannel.invokeMethod(
+        'shareMethod',
+        {'message': message},
```

Comment:
if u share only a string it redundant to use map. U can just pass string to the native platform